referencia/onepay/README.md

```diff
@@ -1117,3 +1117,21 @@ Y en la clase Java del Activity o Fragment en donde estás agregando el componen
 QROnepayView imageViewQrCode = inflatedView.findViewById(R.id.qr_imageView);
 imageViewQrCode.setOtt(ott);
 ```
+
+### Dibujar QR para implementar modalidad Cortafilas en iOS
+
+Para integrar este componente, puedes hacerlo por una de las dos formas:
+
+* Programáticamente. Dentro la inicialización de tu UIViewController, inicializa la clase QROnepayView y agrégalo a tu vista de la siguiente manera:
+
+```swift
```

Comment:
I'm afraid the swift snippets won't be shown on the code panel as there is no "swift" tab. 

Take a look at other parts of the onepay reference document to see how to hack your way into show mobile code snippets

Reply:
(That might apply to other stuff you have added to the reference document before and was already reviewed by others, so make sure you apply this change to any mobile-sdk snippet)

Reply:
@leosoto ¡Great!, fixed in 8fe1628